### ISSUE TITLE
Analytic tags in sales form still show up even though analytic accounting is disabled for sales.

### ISSUE THREAD

**Issue by icoro** [odoo/odoo#24722]
Impacted versions: V11

Steps to reproduce:
(1) Activate analytic accounting in sales
(2) Go to sales form and analytic tags appear in form.
(3) Disable analytic accounting in sales
(3) Go to sales form and analytic tags disappear, which is the correct behaviour
(4) Now activate analytic accounting for purchases
(5) Go to sales form and analytic tags always appear even if I disable analytic accounting for sales.

So in summary, the analytic tags in sales form appear always as long as purchase analytic accounting is enabled. It doesn't matter if I disable analytic accounting for sales, the tags still show up in sales form. Please advice and confirm this bug.
